src/index.js

```diff
@@ -53,45 +54,56 @@ function createBabelPluginFactory(customCallback = returnObject) {
 			externalHelpersWhitelist,
 			include,
 			runtimeHelpers,
+			transformGenerated,
 			...babelOptions
 		} = unpackOptions(pluginOptions);
 
+		if (transformGenerated) {
+			return {
+				name: 'babel-output',
```

Comment:
Not sure if this should receive a different name. Its still the same plugin - just uses a different strategy for transformation.

Reply:
My motivation was to have different names for the case when we apply this plugin twice, once on the input and once on the output. But as we are not using the plugin cache, I guess this will not matter and should not cause warnings. I will change it.

Reply:
I guess I will also add an example of using the plugin twice to the README.

Reply:
There is an old PR for using the cache - https://github.com/rollup/rollup-plugin-babel/pull/235 . If this matter for caching I'm not against a different name - could you maybe take a look at that PR to assess what would have to be done to finish it?

Reply:
Actually that plugin appears to utilise a "cacheKey", which circumvents the problem of duplicate names. This one needs to be adapted but it could actually help speed up watch mode. Would be worth another look once this is merged. The open remaining question was how to create a good cacheKey that reflects all involved plugins and version numbers. And this is non-trivial as the plugins could change on a per-file basis while a cacheKey would be constant for the plugin.

Reply:
Ok, resolving this comment as out of the scope of this PR. We can circle back to this after this gets merged in.